apps/bixarena/app/src/model/models_api.py

```diff
@@ -0,0 +1,124 @@
+"""
+Simple FastAPI endpoint to serve model data
+Single endpoint: GET /models
+Includes database access functions - no external dependencies
+"""
+
+import subprocess
+from fastapi import FastAPI, HTTPException
+from typing import Any
+import uvicorn
+
+app = FastAPI(title="BixArena Models API", version="1.0.0")
```

Comment:
It's an anti-pattern to have two apps in the same Nx project in the monorepo. The REST API should have its own project. I scheduled some time for us to meet tomorrow to implement this as part of the existing Java REST API.